src/scripts/videohandler.ts

```diff
@@ -0,0 +1,25 @@
+((require as unknown) as typeof define)(["playbackManager", "events"], function (playbackManager, events) {
```

Comment:
Same for this, use the pre-made typings.

Reply:
Same conflict here